sql/src/main/java/io/crate/metadata/settings/CrateSettings.java

```diff
@@ -284,9 +284,39 @@ Settings settings() {
         return referenceImplementationTree;
     }
 
+    private void buildGroupSettingReferenceTree(String prefix, Map.Entry<String, Settings> SettingEntry, Map<String, ReferenceImplementation> referenceMap) {
```

Comment:
I'd appreciate to move this method under the `buildReferenceTree` method which is using this one